csrankings-e.csv

```diff
@@ -69,6 +69,7 @@ Eduard C. Dragut,Temple University,https://cis.temple.edu/~edragut,NOSCHOLARPAGE
 Eduard Constantin Dragut,Temple University,https://cis.temple.edu/~edragut,NOSCHOLARPAGE
 Eduard Eiben,Royal Holloway University of London,https://pure.royalholloway.ac.uk/portal/en/persons/eduard-eiben(9ee8fd8f-cbc6-4365-bda4-79850f62e36d).html,VzCxpJcAAAAJ
 Eduard Gröller,TU Wien,https://www.cg.tuwien.ac.at/staff/EduardGroeller.html,eBsjFlIAAAAJ
+Eduard Hovy,University of Melbourne,https://cis.unimelb.edu.au/research/artificial-intelligence/Natural-Language-Processing#:~:text=Eduard%20Hovy,PUFxrroAAAAJ

```

Comment:
Paper from 2022 still has him with a CMU affiliation, as does his Google Scholar page (also, the home page should not contain the text highlighting in the URL). This web page does not indicate that he is a faculty member. Please re-open with evidence that he meets the inclusion eligibility requirements.